crates/ark/src/variables/r_variables.rs

```diff
@@ -66,6 +67,9 @@ pub struct RVariables {
     /// thread. Tracked in https://github.com/posit-dev/positron/issues/1812
     current_bindings: RThreadSafe<Vec<Binding>>,
     version: u64,
+    /// Whether to always show the .Last.value in the Variables pane, regardless
+    /// of the value positron.show_last_value
+    show_last_value: bool,
```

Comment:
I have a preference towards avoiding booleans where possible for call site readability

```rust
last_value: LastValue
```

```rust
enum LastValue {
  Show,
  Hide
}
```

In particular I think it really helps with

```rust
        // Start with default settings
        Self::start_with_config(env, comm, comm_manager_tx, false);
```

vs

```rust
        // Start with default settings
        Self::start_with_config(env, comm, comm_manager_tx, LastValue::Hide);
```

Reply:
Yes, I usually do this too but was lazy here. 😅 Updated.